src/main/java/com/ezylang/evalex/functions/datetime/DateTimeFormatFunction.java

```diff
@@ -0,0 +1,45 @@
+/*
+  Copyright 2012-2022 Udo Klimaschewski
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+package com.ezylang.evalex.functions.datetime;
+
+import com.ezylang.evalex.Expression;
+import com.ezylang.evalex.data.EvaluationValue;
+import com.ezylang.evalex.functions.AbstractFunction;
+import com.ezylang.evalex.functions.FunctionParameter;
+import com.ezylang.evalex.parser.Token;
+import java.time.LocalDateTime;
+import java.time.ZoneOffset;
+import java.time.format.DateTimeFormatter;
+
+@FunctionParameter(name = "value", isVarArg = true)
+public class DateTimeFormatFunction extends AbstractFunction {
+  @Override
+  public EvaluationValue evaluate(
+      Expression expression, Token functionToken, EvaluationValue... parameterValues) {
+    String formatted;
+    if (parameterValues.length < 2) {
+      formatted =
+          LocalDateTime.ofInstant(parameterValues[0].getDateTimeValue(), ZoneOffset.UTC).toString();
```

Comment:
I'm a bit unsure about the fixed UTC offsets here and in other places, woudn't the current time zone be a better choice?

Reply:
I completely agree with you. I'm also not convinced that using fixed UTC offsets is the best approach. In fact, I've been thinking that it would be more flexible if we could have a configuration option to force a specific time zone, with the default being the system time zone. This way, users could set their preferred time zone and not have to worry about inconsistencies or inaccuracies caused by using fixed offsets. What do you think?

Reply:
Having a configuration property for the default time zone sounds good. If not set explicitly, the system default time zone could be used.